src/Components/Dialogs/GetPerson/getPerson.jsx

```diff
@@ -0,0 +1,14 @@
+
+import getElement from './GetElement/getElement';
+
+const GetPerson = (props) => {
+  const { items } = props;
+  const listItems = Object.keys(items);
+  const result = getElement(listItems, items);
```

Comment:
See the comment in getMessage component.

Reply:
Same as above